TravelAgency-spring-mvc/src/main/webapp/WEB-INF/jsp/customer/edit.jsp

```diff
@@ -40,17 +41,21 @@
                 </tr>
                 <tr>
                     <td><form:label path="password"><f:message key="customer.password"/>: </form:label></td>
-                    <td><form:input path="password"/></td>
+                    <td><form:input type="password" path="password"/></td>
                 </tr>
-                <br/> 
+                <br/>
```

Comment:
You can have all the <br/> tags in 1 line if you want.  I think it looks better. It's up to you tho.